provider/lxd/config.go

```diff
@@ -65,3 +73,11 @@ func (c *environConfig) validate() error {
 	// There are currently no known extra fields for LXD
 	return nil
 }
+
+func (c *environConfig) project() string {
+	project := c.attrs["project"]
+	if project == nil {
+		return ""
+	}
+	return project.(string)
```

Comment:
Do we need a type check here?

Reply:
It's validated on the way in. Other providers like ec2 return in the same fashion.